java/feathub-connectors/flink-connectors/flink-connector-redis/src/main/java/com/alibaba/feathub/flink/connectors/redis/lookup/RedisLookupFunction.java

```diff
@@ -87,6 +94,23 @@ public RedisLookupFunction(ReadableConfig config, ResolvedSchema schema) {
         this.featureFieldIndices = getFeatureFieldIndices(schema, config);
         this.physicalKeyFieldIndices = getPhysicalKeyFieldIndices(schema, featureFieldIndices);
         this.fieldDataTypes = schema.getColumnDataTypes().toArray(new DataType[0]);
+        if (config.get(HASH_FIELDS) == null || this.featureFieldIndices.length > 1) {
```

Comment:
Should we also support `HASH_FIELDS` when the Redis table only has more than one feature column?